vikitaotiz/iReporter-Api
# [#162298092] Get All Red Flag Incidences

**What does this PR do?**
This PR creates get API endpoint for all Red Flag incidences

**Description of Task to be completed:**
User should be able to view all Red Flag incidences

**How to manually test**
$ pip install virtualenv
NB This install virtualenv Globally(virtual environment allows you to create independent env isolated from your actual physical machine)
$ virtualenv -p python3 env
$ source env/bin/activate
Clone this repository: $ https://github.com/vikitaotiz/iReporter-Api.git
navigate into the repository after cloning
$ git checkout  ft-get-all-red-flag-records-162298092 to switch from master to this branch
$ pip install -r requirements.txt
$ pytest

Any background information
User should be able to view all Red Flag incidences from this endpoint

**Relevant pivotal story**
#162298092

**Screenshot**
![get-all](https://user-images.githubusercontent.com/42869046/49264540-c0e36a80-f45f-11e8-89b9-b1c62ba572ac.PNG)
